Automattic/wp-calypso
# Editor: Removes editor deprecation flags in favor of user deprecation group

#### Changes proposed in this Pull Request

Removes the `editor/after-deprecation` and `editor/before-deprecation` feature flags, in favor of the user deprecation group.

#### Testing instructions

Before deprecation

1. If you've previously dismissed the editor deprecation notice, clear Calypso state in browser storage (IndexDB)
1. Visit the a post with the Calpyso editor (`/post/{site}/{post_id}?set-editor=classic`)
1. You should see a notice about "The new WordPress editor is coming." in a modal
1. Dismiss the modal and reload the page, you should _not_ see the modal again
1. Click on the inline help in the bottom right of the screen, you should see a link for "Switch to Block Editor"

---

After deprecation

Set a user attribute for your user using `wpsh` on a sandbox `$ update_user_attribute( USER_ID, 'editor_deprecation_group', true )`

1. Clear Calypso state in browser storage (IndexDB) so that Calypso doesn't remember you dismissed the notice
1. Visit the a post with the Calpyso editor (`/post/{site}/{post_id}?set-editor=classic`)
1. You should _not_ see a notice about the deprecation
1. Click on the inline help in the bottom right of the screen, you should _not_ see a link for "Switch to Block Editor"
